cc/testing/evaluation/statisticaltester/StatisticalTesterSum.java

```diff
@@ -0,0 +1,248 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.google.privacy.differentialprivacy.testing.StatisticalTestsUtil;
+import java.io.*;
+import java.lang.*;
+import java.util.*;
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/** Runs the Statistical Tester on differentially private sums with
+insufficient noise. Records the results in an output text file.
+*/
+public class StatisticalTesterSum {
```

Comment:
Many comments on StatisticalTesterCount apply here as well. More generally, the similarity of StatisticalTesterCount, StatisticalTesterMean, and StatisticalTesterSum suggest that we should try to design-out their similarities such that there isn't so much of repeated code.